R/pkg/R/context.R

```diff
@@ -126,13 +126,13 @@ parallelize <- function(sc, coll, numSlices = 1) {
   if (numSlices > length(coll))
     numSlices <- length(coll)
 
-  sizeLimit <- .Machine$integer.max - 10240 # Safe margin bellow maximum allocation limit
+  sizeLimit <- as.numeric(
```

Comment:
agreed, probably not a big deal, an user could set `spark.r.maxAllocationLimit` to `0.01` though.